neqo-transport/src/cc/tests/new_reno.rs

```diff
@@ -22,9 +20,7 @@ use crate::{
     rtt::RttEstimate,
 };
 
-const IP_ADDR: IpAddr = IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0));
-const PTO: Duration = Duration::from_millis(100);
-const RTT: Duration = Duration::from_millis(98);
+const PTO: Duration = RTT;
```

Comment:
I assume it is fine that `PTO` is now equal to `RTT`, while previously it was `RTT - 2`.

Reply:
It doesn't seem to cause a test failure 🤷